.storybook/stories/KvButton.stories.js

```diff
@@ -20,6 +20,11 @@ export const DefaultSmallest = () => ({
 	template: '<kv-button class="smallest">Button Smallest</kv-button>'
 });
 
+export const Rounded = () => ({
+	components: { KvButton },
+	template: '<kv-button class="rounded">Button</kv-button>'
```

Comment:
#sadpanda  I wish design would choose whether our buttons are rounded or our buttons are not rounded. I don't see a compelling reason to have 2 options.

Reply:
It is a little frustrating but overall I think what @emuvente has done here and has started could help us chart a way forward for adding in new styles alongside our existing styles too.